<!--Wed Apr 24 19:24:04 2013-->
  <configuration>
  <property>
    <name>dfs.safemode.threshold.pct</name>
    <value>1.0f</value>
  </property>
  <property>
    <name>dfs.data.dir</name>
    <value>/mnt/mnt0/hadoop/dfs/data,/mnt/mnt1/hadoop/dfs/data,/mnt/mnt2/hadoop/dfs/data,/mnt/mnt3/hadoop/dfs/data</value>
  </property>  
  <property>
    <name>dfs.namenode.handler.count</name>
    <value>40</value>
  </property>
  <property>
    <name>dfs.block.size</name>
    <value>536870912</value>
  </property>
  <property>
    <name>dfs.name.dir</name>
    <value>/mnt/mnt0/hadoop/dfs/namenode,/mnt/mnt1/hadoop/dfs/namenode,/mnt/mnt2/hadoop/dfs/namenode,/mnt/mnt3/hadoop/dfs/namenode</value>
  </property>
  <property>
    <name>dfs.replication</name>
    <value>3</value>
  </property>
  <property>
    <name>dfs.datanode.du.reserved</name>
    <value>1</value>
  </property>
  <property>
    <name>dfs.replication.max</name>
    <value>10</value>
  </property>
  <property>
    <name>dfs.datanode.failed.volumes.tolerated</name>
    <value>0</value>
  </property>
  <property>
    <name>dfs.webhdfs.enabled</name>
    <value>false</value>
  </property>
  <property>
    <name>dfs.datanode.max.xcievers</name>
    <value>4096</value>
  </property>
  <property>
    <name>dfs.access.time.precision</name>
    <value>0</value>
  </property>
  <property>
    <name>dfs.http.address</name>
    <value>0.0.0.0:50070</value>
  </property>
  <property>
    <name>dfs.datanode.http.address</name>
    <value>0.0.0.0:50075</value>
  </property>
    <property>
    <name>dfs.datanode.ipc.address</name>
    <value>0.0.0.0:8010</value>
  </property>
  <property>
    <name>dfs.client.read.shortcircuit</name>
    <value>true</value>
  </property>
  <property>
    <name>dfs.block.local-path-access.user</name>
    <value>ubuntu</value>
  </property>
</configuration>
